source/one-or-more.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+Either matches an array containing only `Value`, or `Value` itself, not in an array.
+
+@example
+```
+import {OneOrMore} from 'type-fest';
+
+type ID = number;
+type OneOrMoreIDs = OneOrMore<ID>;
+
+const multipleIDs: OneOrMoreIDs = [1234, 2468];
+const oneID: OneOrMoreIDs = 1234;
+```
+*/
+export type OneOrMore<Value> = Value | readonly Value[];
```

Comment:
Non-blocking, just curious: Why is it a `readonly` array?

Reply:
To be honest I just went by what this dude said. Didn't really give it much thought. :stuck_out_tongue: Do you think it'd be better as *not* read-only?
> https://github.com/sindresorhus/type-fest/issues/66#issuecomment-584741822

Reply:
Lodash seems to be using the `Many` type only for typing function parameters, so it makes sense to have the `readonly` params.

Hard to think of any other application of this type in real world, so I guess this won't cause any issues.

Reply:
The value of this type is accepting input; it's rare that functions return a single item or a list since they can just always return a list.
Using `readonly` allows read-only arrays to be passed in, since `ReadonlyArray` is a subset of `Array`.
```ts
const doThing = (params: OneOrMore<number>) => {};

// both array and readonly array work
const items = [] as readonly number[];
const things = [] as number[];
doThing(items);
doThing(things);
```

Without this you would have to work around it:
```ts
doThing([...items]);
```